app/src/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -564,7 +565,7 @@ public void onClick(View v) {
                 if (!"done".equals(v.getTag())) {
                     FormEntryActivity.this.showNextView();
                 } else {
-                    FormEntryActivity.this.triggerUserFormComplete();
+                    saveDataToDisk(EXIT, true, getDefaultFormTitle(), false);
```

Comment:
Hey,

So we used the old statement (triggerUserFormComplete) to denote the distinction between the form saving for an arbitrary reason with user input that means "I want the form to exit", which has some important distinctions, UI wise. I don't think they're relevant without some new UI stuff, but I think it would be good to maintain them